4주차/login-signup/src/Login.jsx

```diff
@@ -0,0 +1,118 @@
+import { useState, useEffect } from "react";
+import styled from "styled-components";
+import "./App.css";
+import axios from "axios";
+import { Link, Navigate, useNavigate } from "react-router-dom";
+import { createPortal } from "react-dom";
+import {
+  MainBox,
+  Header,
+  Section,
+  Sections,
+  SectionInput,
+  SectionTitle,
+} from "./style";
+
+export default function Login() {
+  const [username, setUsername] = useState("");
+  const [password, setPassword] = useState("");
+
+  const [showToast, setShowToast] = useState(false);
+  const [toastMessage, setToastMessage] = useState("");
+
+  const navigate = useNavigate();
+
+  const handleUserName = (e) => {
+    setUsername(e.target.value);
+  };
+
+  const handlePassword = (e) => {
+    setPassword(e.target.value);
+  };
+
+  const handleLogin = () => {
+    axios
+      .post(import.meta.env.VITE_BASE_URL + "/api/v1/members/sign-in", {
+        username: username,
+        password: password,
+      })
```

Comment:
```suggestion
      .post(`${import.meta.env.VITE_BASE_URL}/api/v1/members/sign-in`, {
        username,
        password,
      })
```
템플릿 리터럴과 단축 프로퍼티 사용하기!